draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1507,7 +1509,7 @@ or around the world is not required.
 
 Although true global uniqueness is impossible to guarantee without a shared
 knowledge scheme, a shared knowledge scheme is not required by UUID to provide
-uniqueness guarantees.
+uniqueness for practical implementation purposes.
```

Comment:
Same logic as earlier - we can't guarantee uniqueness in all cases, we're basically saying it's good enough for practical purposes.